lib/logging.properties

```diff
@@ -37,3 +37,5 @@ timeseries.level=OFF
 timeseries.org.jitsi.videobridge.cc.vp8.level=ALL
 timeseries.useParentHandlers = false
 timeseries.handlers = java.util.logging.FileHandler
+
+org.jitsi.xmpp.mucclient.MucClientManager.level=SEVERE
```

Comment:
Let me fix the logging instead please (PR coming)

Reply:
I had removed this change.